rest_framework_extensions/mixins.py

```diff
@@ -57,13 +61,11 @@ def get_queryset(self):
 
     def filter_queryset_by_parents_lookups(self, queryset):
         parents_query_dict = self.get_parents_query_dict()
+
         if parents_query_dict:
-            try:
```

Comment:
i can see it could be more idiomatic, can you adjust the related unit and functional tests lil bit?

Reply:
Hello, good day.

I have changed how I declared the class methods to make them more idiomatic.

Regarding unit and functional test, I didn't find any relevant changes to make.

Reply:
https://github.com/chibisov/drf-extensions/tree/master/tests_app/tests